docs/api/topics/firefox_os_addons.rst

```diff
@@ -116,6 +116,18 @@ Search
     :status 200: successfully completed.
 
 
+Delete
+------
+
+.. _addon-delete:
+
+.. http:delete:: /api/v2/extensions/extension/(int:id)|(string:slug)/
+
+    .. note:: Requires authentication. Only works on your own Add-ons.
+
+    Delete an add-on. This action is irreversible.
```

Comment:
It's not. Having it causes subtle problems, so as long as no story requires it, I'd prefer not having it for the moment. When/If we add it, I'll change the docs.